docs/binary/quickstart.rst

```diff
@@ -0,0 +1,125 @@
+Quickstart Guide
+================
+
+The primary input language for cvc5 is
+`SMT-LIB v2 <http://smtlib.cs.uiowa.edu/language.shtml>`_.
+We give a short explanation of the SMT-LIB v2 equivalent of the quickstart
+example here.
+
+First, we set the logic..
+The simplest way to set a logic for the solver is to choose "ALL".
+This enables all logics in the solver.
+Alternatively, ``"QF_ALL"`` enables all logics without quantifiers.
+To optimize the solver's behavior for a more specific logic,
+use the logic name, e.g. ``"QF_BV"`` or ``"QF_AUFBV"``.
+
+.. literalinclude:: ../../examples/api/smtlib/quickstart.smt2
+     :language: smtlib
+     :lines: 1
+
+We will ask the solver to produce models and unsat cores in the following,
+and for this we have to enable the following options.
+Furthermore, we enable incremental solving so that we can use the
+``(reset-assertions)`` command later on.
+
+.. literalinclude:: ../../examples/api/smtlib/quickstart.smt2
+     :language: smtlib
+     :lines: 2-6
+
+Now, we create two constants ``x`` and ``y`` of sort ``Real``.
+Notice that these are *symbolic* constants, but not actual values.
+
+.. literalinclude:: ../../examples/api/smtlib/quickstart.smt2
+     :language: smtlib
+     :lines: 8-10
+
+We define the following constraints regarding ``x`` and ``y``:
+
+.. math::
+
+  (0 < x) \wedge (0 < y) \wedge (x + y < 1) \wedge (x \leq y)
+
+We construct the required terms and assert them as follows:
```

Comment:
I think this line would be more appropriate if we would have used some `define-fun`s, but we don't. So how about:

```suggestion
We assert them as follows. Notice that in SMT-LIB v2, terms are written in prefix notation, e.g., we write `(+ x y)` instead of `(x + y)`.
```